content/sensu-enterprise/3.1/guides/using-ec2.md

```diff
@@ -0,0 +1,238 @@
+Note the ARN for this policy, needed for step 3
+
+2. Create the IAM role
+{{< highlight shell >}}
+aws iam create-role --role-name sensu-enterprise-integration --assume-role-policy-document file://sensu-ec2-iam-role.json{{< /highlight >}}
+
+3. Attach policy to role
+{{< highlight shell >}}
+aws iam attach-role-policy --role-name sensu-enterprise-integration --policy-arn "arn:aws:iam::11111111111:policy/sensu-ec2-iam-policy"{{< /highlight >}}
+
+Now that we've created a policy and a role for the integration, we'll need to attach it to our Sensu Enterprise instance. You can do this during step 3 of creating any instance via the EC2 console:
+
+![ec2_role_create][10]
+
+Now that we have the policy and role created, and the role attached to an instance, let's see this in action on a test instance:
+
+![ec2_test_gif][11]
+
+## Cross-Account IAM Roles
+
+### Definitions
+
+Trusting account -- an AWS account where Sensu Clients are running. An
+IAM 'read-only' role is created here allowing ec2:DescribeInstances
+action, with a policy that allows the trusted account's instance role
+to assume the read-only role.
+
+For the purposes of this document:
+* The AWS account number for the trusting account is assumed to be `111111111111`
+* The AWS CLI is assumed to be configured with a profile `trusting`
+
+Trusted account -- an AWS account where Sensu Enterprise is running. An
+EC2 instance is created here with an instance role attached allowing
+the instance to assume a role in the trusting account.
+
+For the purposes of this document:
+* The AWS account number for the trusted account is assumed to be `222222222222`
+* The AWS CLI is assumed to be configured with a profile `trusted`
+
+### Setup
+
+You can download the following files to follow along:
+
+* [Cross Account RO Policy][12]
+* [Cross Account RO Role][13]
+* [Cross Account Assume Role Policy][14]
+* [Cross Account Assume Role Policy Role][15]
+
+#### Trusting account
+
+1. Create 'read-only' IAM policy
+{{< highlight shell >}}
+aws iam create-policy --profile trusting --policy-name cross-account-readonly --policy-document file://cross-account-readonly-policy.json
+{{< /highlight >}}
+Note the ARN for this policy, needed for step 3
+
+2. Create 'cross-account' IAM role
+{{< highlight shell >}}
+aws iam create-role --profile trusting --role-name cross-account-readonly --assume-role-policy-document file://cross-account-readonly-role.json
+{{< /highlight >}}
+
+3. Attach policy to role
+{{< highlight shell >}}
+aws iam attach-role-policy --profile trusting --role-name cross-account-readonly --policy-arn "arn:aws:iam::111111111111:policy/cross-account-readonly"
+{{< /highlight >}}
+
+#### Trusted account
+
+1. Create 'cross-account-assumerole' IAM policy
+{{< highlight shell >}}
+aws iam create-policy --profile trusted --policy-name cross-account-assumerole --policy-document file://cross-account-assumerole-policy.json
+{{< /highlight >}}
+Note the ARN for this policy, needed for step 3
```

Comment:
While I don't know if an official NOTE is needed here, it looks odd on the website.